devel.module

```diff
@@ -1411,8 +1411,8 @@ function devel_query_table($queries, $counts) {
   $version = devel_get_core_version(BACKDROP_VERSION);
   $header = array ('ms', '#', 'where', 'ops', 'query', 'target');
   $i = 0;
-  $api = config->get('api_url');
-  $slow_query_limit = config->get('query_slow_limit');
+  $api = config_get('api_url');
```

Comment:
This actually needs to be:

```suggestion
  $api = $config->get('api_url');
```

...either that, or you'd need to remove the `$config = config('devel.settings');` a few lines above, and have this be `config_get('devel.settings', 'api_url');`. If you did that, you'd then also have to change every occurrence of `config_get()` in this function, in order to pass the config file parameter.

Reply:
The original code was missing the dollar sign before `config->get()`, which didn't make sense to me, so I thought it was just confusion with the `config_get()` function. 

Also when you give me either-or option I feel I shouldn't be one to decide, because I don't own the original code. Please tell me what's the best course here. 

Has anyone did run tests with `$api = $config->get('api_url');`? Does that work correctly and was that `$` just forgotten?

Reply:
> Does that work correctly and was that `$` just forgotten?

Yup, that's what it was. Sorry, rookie mistake on my part 😅 ...@docwilmot gave a really good explanation in https://github.com/backdrop-contrib/devel/pull/86#issuecomment-549581237 about how things work.